src/Microsoft.FeatureManagement/FeatureManagementError.cs

```diff
@@ -9,18 +9,23 @@ namespace Microsoft.FeatureManagement
     public enum FeatureManagementError
     {
         /// <summary>
-        /// A feature filter that was listed for feature evaluation was not found.
+        /// A feature filter configured for the feature being evaluated is an ambiguous reference to multiple registered feature filters.
         /// </summary>
-        MissingFeatureFilter,
+        AmbiguousFeatureFilter,
 
         /// <summary>
-        /// A feature filter configured for the feature being evaluated is an ambiguous reference to multiple registered feature filters.
+        /// A configuration error is present in the feature management system.
         /// </summary>
-        AmbiguousFeatureFilter,
+        InvalidConfiguration,
```

Comment:
It seems like every error here could be classified as an `InvalidConfiguration`. In a way it seems like a catch all error-code.

Thinking -- how can we be more specific with the error code to the problem we're adding it for.

Reply:
Removed Invalid Configuration and added 

```
        /// <summary>
        /// A configuration had conflicting values.
        /// </summary>
        Conflict,

        /// <summary>
        /// The given value was invalid.
        /// </summary>
        InvalidValue
```